### ISSUE TITLE
Contradiction regarding rust version?

### ISSUE THREAD

**Issue by c-git** [emilk/eframe_template#110]
In the [Testing Locally](https://github.com/emilk/eframe_template#testing-locally) section of the readme, it says 

> Make sure you are using the latest version of stable rust by running `rustup update`

but the repo also includes a toolchain file. Wouldn't that mean that running `rustup update` wouldn't matter?

**Comment by gcp** [comment]
There's a similar oddity in that ```rust.yml``` uses ```toolchain: stable``` everywhere except: https://github.com/emilk/eframe_template/blob/master/.github/workflows/rust.yml#L99

**Comment by c-git** [comment]
I feel these are likely just oversights not sure what's the best way to address them as I think they are already a bit overloaded with PRs

**Comment by mowdl** [comment]
For me, running `cargo install --locked trunk` would not work inside the project folder because trunk needs a newer version of rust.

You need the latest version to install `trunk` but to compile the app you use the version specified by the `rust-toolchain` file.

**Comment by YertleTurtleGit** [comment]
I'm facing the same confusion. I was also wondering if this causes my IDE (VS Codium) to not properly initialize clippy:
```
cargo check failed to start: Cargo watcher failed, the command produced no valid metadata (exit code: ExitStatus(unix_wait_status(256))):

error: 'cargo-clippy' is not installed for the custom toolchain '/home/user/.rustup/toolchains/1.72.0-x86_64-unknown-linux-gnu'.

note: this is a custom toolchain, which cannot use `rustup component add`

help: if you built this toolchain from source, and used `rustup toolchain link`, then you may be able to build the component with `x.py`
```
Any ideas about how to solve this or if this is even related?

**Comment by c-git** [comment]
Yes that problem is caused by using the different version of rust. Just remove the `rust-toolchain` file and you should be good to go.

**Comment by c-git** [comment]
After having had time to think about this. I think updating the readme is the right approach because the toolchain file gets updated each release.
